common/openapi.yml

```diff
@@ -463,6 +508,11 @@ components:
       type: string
       format: string
 
+    PipelineJobOutput:
```

Comment:
im assuming this will get expanded as we figure out what things we want to return to the user upon a completed pipeline job?

Reply:
yes, exactly right